oximeter/oximeter/schema/switch-data-link.toml

```diff
@@ -228,6 +228,18 @@ versions = [
     { added_in = 1, fields = [ "port_id", "lane", "link_id" ] }
 ]
 
+[fields.kind]
+type = "string"
+description = "The kind or class of the data link (examples: physical-device, gateway, switch-port)"
+
+[fields.model]
+type = "string"
+description = "Model type of the link, indicating what implements or enables the link itself (examples: TF2, tfportd, STM32H753, opte)"
```

Comment:
Let's make these match the comments in the other TOML file, however you end up changing those.